src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -93,8 +95,15 @@ private void InitializeColors()
             CodeAnalysisColors.AccentBarColorKey = EnvironmentColors.FileTabInactiveDocumentBorderEdgeBrushKey;
         }
 
-        protected override void LoadComponentsInUIContext(CancellationToken cancellationToken)
+        protected override async Task LoadComponentsInUIContextAsync(CancellationToken cancellationToken)
         {
+            await GetServiceAsync(typeof(SVsTaskStatusCenterService)).ConfigureAwait(true);
```

Comment:
❗️ Need an explicit `SwitchToMainThreadAsync` before the call to `GetService<MiscellaneousFilesWorkspace>`. For this initialization method, it likely would not hurt to place the call as the first operation of the method.